src/openloco/windows/townwnd.cpp

```diff
@@ -664,31 +658,6 @@ namespace openloco::ui::windows::town
             self->widgets[common::widx::panel].bottom = self->height - 1;
         }
 
-        static void on_resize(window* self)
-        {
```

Comment:
What is the reasoning behind this change

Reply:
This is the common::on_resize function from the town window. However, it looks like the code is shared not just between tabs in the town window, but other windows as well — such as the company window. I have therefore moved it to a common window function instead.

Reply:
This is also an event though. What will the town window call on the on resize function call? I'm not sure you can just remove the function completely.

Reply:
Despite appearances, it's not — all event instances called this common function. They no longer have to if the functionality is moved to `window::set_size`.

Perhaps viewing https://github.com/OpenLoco/OpenLoco/pull/414/commits/fb87d8d31dcade51e76f87c96c72e831e45a0dc0 on its own will clear up what the change is doing?

Reply:
But what if other windows want to call set size but aren't expecting the extra bounds checking? Or is set_size a function we have created?

Reply:
Ah! It's very much our own, yes. There's two with the same name, but a different signature: one for setting the size, and one for setting size along with min/max size. This one is the latter.

Reply:
Okay this is fine then.